eth/downloader/downloader.go

```diff
@@ -1047,7 +1047,7 @@ func (d *Downloader) processHeaders(origin uint64) error {
 		if rollback > 0 {
 			curBlock := d.core.CurrentBlock().NumberU64()
 			log.Warn("Rolled back chain segment",
-				"block", fmt.Sprintf("%d->%d", curBlock), "reason", rollbackErr)
+				"block", fmt.Sprintf("%d->%d", curBlock, rollbackErr), "reason")
```

Comment:
Neither of these print statements look correct, lol. I don't think this is the linters fault, but just want to check with you first